app/models/like.rb

```diff
@@ -0,0 +1,10 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: :author_id
+  belongs_to :post
+
+  after_save :update_likes_counter
+
+  def update_likes_counter
```

Comment:
- [ ] OPTIONAL
Following best practices `update_likes_counter` should be marked as private after implementing the `after_save` method.